installers/home/radiasoft-download.sh

```diff
@@ -4,5 +4,6 @@
 #
 home_main() {
     install_url biviosoftware/home-env
-    install_script_eval install.sh
+    bivio_home_env_ignore_git_dir_ownership=${vagrant_dev_ignore_git_dir_ownership:+1}  \
```

Comment:
That's not usually how we cascade, but I guess it works. Usually it's a pure value copy:

```sh
bivio_home_env_ignore_git_dir_ownership=${vagrant_dev_ignore_git_dir_ownership:-}
```

I wonder if the cascade is necessary (see below).

Reply:
9b21050